### DISCUSSION TITLE
In LiveCharts2, how can I achieve gradients?

### DISCUSSION THREAD

**Discussion by xiaotebie** [Live-Charts/LiveCharts2#1161] [Q&A]
In LiveCharts, I achieved the gradient effect using the following code:
Code:
```
LineSeries series1 = new LineSeries
            {
                Title = "Value1",
                Values = new ChartValues<double>(),
                Fill = new LinearGradientBrush
                {
                    StartPoint = new Point(0, 0),
                    EndPoint = new Point(0, 1),
                    GradientStops = new GradientStopCollection
                    {
                         new GradientStop(Color.FromArgb(0x55, 0x2B, 0xED, 0xF1), 0),
                        new GradientStop(Colors.Transparent, 1)
                    }
                },
                Stroke = (Brush)new BrushConverter().ConvertFrom("#FF2BEDF1") ?? Brushes.LightBlue,               
            };

```
Resulting Chart in livecharts1:
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/18098862/9b1f76bf-946d-40e5-a7a1-6dd5897e596b)

In LiveCharts2, how can I achieve the same gradient effect?

**Comment by beto-rodriguez** [comment]
Hi, there is a gradients article in the site: https://lvcharts.com/docs/WPF/2.0.0-beta.920/samples.design.linearGradients

**Reply by xiaotebie** [reply]
Thank you very much, the problem has been solved!
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/18098862/5f39c448-9adc-4abd-b944-d5378b0668f9)

**Reply by mcash95** [reply]
I am looking to achieve the same gradient effect with a Gauge, and PieCharts. Is this possible?
